build-pipelines/scripts/yaml-integration-tests.sh

```diff
@@ -44,6 +44,8 @@ do
        fi                                                                
     done
   if [[ -n "${envId}" && -n "${tenantId}" && -n "${domain}" && -n "${testPlanFile}" && -n "${outputDir}" ]]; then     # null checks on args
-    dotnet run -- -e ${envId} -t ${tenantId} -d ${domain} -i ${testPlanFile} -o ${outputDir};  
+    dotnet run -- -e ${envId} -t ${tenantId} -d ${domain} -i ${testPlanFile} -o ${outputDir};
+    domain = "${domain}&PAOverrideFGRollout.ForceOnePlayer=true";
```

Comment:
if the domain looks like `make.test.powerapps.com` we may need to append `?PAOverrideFGRollout.ForceOnePlayer=true ` instead

Reply:
It already has &source=testengine appended to it, right?

Reply:
It looks like this, how its processed 
<img width="638" alt="image" src="https://user-images.githubusercontent.com/98551644/216468600-6f8ab862-bc74-44fb-bc5d-02abff31c18d.png">

Would need another way to pass these params here. Maybe a code change in Program.cs to accept queryParams as an argument.

Reply:
oh interesting, gotcha; thanks... okay, I'll look into this